bits.go

```diff
@@ -0,0 +1,193 @@
+package iabtcf
+
+import (
+	"fmt"
+	"time"
+)
+
+// //////////////////////////////////////////////////
+// bits
+
+// Bits represents a bitset with some helpers to read int, bool, string and time fields
+//
+// Bits are stored in a byte slice
+// First byte will store the first 8 bits, second byte the next 8 bits, and so on
+//
+// note: the last byte may contain less than 8 bits. Those bits are left aligned.
+type Bits []byte
+
+// HasBit checks if the bit number is set
+//
+// note: number is not the index and it starts at 1.
+func (b Bits) HasBit(number int) bool {
+	value, _ := b.ReadBoolField(number - 1)
+	return value
+}
+
+const (
+	NbBitInByte  = 8
+	LastBitIndex = NbBitInByte - 1
```

Comment:
Need to be exported?  Also `BitMasks` below.

Reply:
✔️ Done